src/test/test-compress.c

```diff
@@ -242,7 +242,7 @@ static void test_lz4_decompress_partial(void) {
         memset(&huge[STRLEN("HUGE=")], 'x', HUGE_SIZE - STRLEN("HUGE=") - 1);
         huge[HUGE_SIZE - 1] = '\0';
 
-        r = sym_LZ4_compress_default(huge, buf, HUGE_SIZE, buf_size);
+        r = sym_LZ4_compress_HC(huge, buf, HUGE_SIZE, buf_size, /* level = */ 0);
```

Comment:
Please use `-1` for consistency with others.

Reply:
this is not `compress_blob` wrapper, so suppose it's better to set 0 here

Reply:
Ah, indeed, then use 1, the level needs to be positive.